src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -283,7 +283,7 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
           <InfoContainer main={main}>
             {displayChannel && (
               <AvatarContainer scalingFactor={scalingFactor}>
-                {isLoading || hasAvatarImgLoaded === false ? (
+                {isLoading || (hasAvatarImgLoaded === false && channelAvatarUrl) ? (
```

Comment:
Now, it's not working :D

I mean, I think I know how to fix that. You could just change this line to: 
```tsx
{isLoading || (hasAvatarImgLoaded === false && channelAvatarUrl !== undefined) ? ( ...
```
This should be working because if the user hasn't set his avatar we're getting `null` instead of `undefined` if I'm not mistaken. 
Also , I think you could move this condition inside your custom hook

Reply:
yeah it makes sense

Reply:
Since this can be used anywhere else I think it's a bit risky to assume the null condition will be true everywhere else so I think I will keep it outside the hook for now